src/main/java/org/alliancegenome/curation_api/services/GeneService.java

```diff
@@ -82,75 +76,41 @@
 	@Override
 	@Transactional
 	public ObjectResponse<Gene> deleteById(Long id) {
-		removeOrDeprecateNonUpdated(id, "Gene DELETE API call");
+		deprecateOrDelete(id, true, "Gene DELETE API call", false);
 		ObjectResponse<Gene> ret = new ObjectResponse<>();
 		return ret;
 	}
 
 	@Override
 	@Transactional
-	public void removeOrDeprecateNonUpdated(Long id, String loadDescription) {
+	public Gene deprecateOrDelete(Long id, Boolean throwApiError, String requestSource, Boolean forceDeprecate) {
 		Gene gene = geneDAO.find(id);
 		if (gene != null) {
-			List<Long> referencingDAIds = geneDAO.findReferencingDiseaseAnnotations(id);
-			Boolean anyReferencingEntities = false;
-			for (Long daId : referencingDAIds) {
-				DiseaseAnnotation referencingDA = diseaseAnnotationService.deprecateOrDeleteAnnotationAndNotes(daId, false, loadDescription, true);
-				if (referencingDA != null) {
-					anyReferencingEntities = true;
+			if (forceDeprecate || geneDAO.hasReferencingDiseaseAnnotations(id) || geneDAO.hasReferencingPhenotypeAnnotations(id) ||
+					geneDAO.hasReferencingOrthologyPairs(id) || geneDAO.hasReferencingInteractions(id) ||
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.OperatorWrapCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'||' should be on a new line.